specimen/wildlife-sounds-soundscapes/eml.xml

```diff
@@ -78,15 +74,16 @@ xsi:schemaLocation="eml://ecoinformatics.org/eml-2.1.1 http://rs.gbif.org/schema
             <onlineUrl>https://www.nlbif.nl</onlineUrl>
             <role>PROCESSOR</role>
         </associatedParty>
-        <pubDate>2023-XX-XX</pubDate>
+        <pubDate>2023-06-30</pubDate>
         <language>en</language>
         <abstract>
-            <para>Xeno-canto (XC) is an online database that provides access to sound recordings of wildlife from around the world. The recordings are shared by a growing community of thousands of recordists from around the world, amateurs and professionals alike. The aim of Xeno-canto is to eventually represent sounds of all animals, meaning of all taxa, to subspecies level, of their complete repertoire, with all of the geographic variability, at all stages of development. </para>
-            <para>The website of Xeno-canto (www.xeno-canto.org) was launched in 2005, and initially focussed on Neotropical birds. The scope of Xeno-canto is gradually widening. It is run by the Xeno-canto Foundation for Nature Sounds in The Netherlands, with support from Naturalis Biodiversity Center. </para>
-           <para>Please note that the Xeno-canto dataset shared on GBIF is a subset of the entire collection that is available on www.xeno-canto.org. Only the recordings by recordists who have given their permission to share recording metadata with GBIF are shared at the moment.</para>
+			<para>This dataset covers the sounds of the Chiroptera sound collection of Xeno-canto (XC). </para> 
+			<para>XC is an online database that provides access to sound recordings of wildlife from around the world. The recordings are shared by a growing community of thousands of recordists from around the world, amateurs and professionals alike. The aim of Xeno-canto is to eventually represent sounds of all animals, meaning of all taxa, to subspecies level, of their complete repertoire, with all of the geographic variability, at all stages of development.</para>
+            <para>The website of Xeno-canto (www.xeno-canto.org) was launched in 2005, and initially focussed on Neotropical birds. The scope of Xeno-canto is gradually widening and the Chiroptera sound collection was started in 2023. XC is run by the Xeno-canto Foundation for Nature Sounds in The Netherlands, with support from Naturalis Biodiversity Center. </para>
+            <para>Please note that the Chiroptera sound dataset shared on GBIF is a subset of the entire collection that is available on https://xeno-canto.org/?gid=3. Only the recordings by recordists who have given their permission to share recording metadata with GBIF are shared here.</para>
         </abstract>
         <keywordSet>
-            <keyword>animalia, wildlife, animalia sounds, wildlife sounds</keyword>
+            <keyword>chiroptera, bats, chiroptera sounds, bat sounds</keyword>
```

Comment:
Idem als boven

Reply:
er is inderdaad iets verwisseld tussen wildlife sounds en bats

Reply:
Heb dit aangepast en getest